base/src/test/resources/success/src/Refparo/Model.aya

```diff
@@ -4,10 +4,10 @@ public open data Unit | unit
 
 -- we cannot eta-contract this, same problem as Cubical Agda
 def coe {A B : Type} (eq : A ↑ = B) (x : A) : B
-  => ↑ primCoe (\y => eq y) 0 x
+  => (\y => eq y).primCoe x
 
 def coeFill {A : Type} {x : A} : x = coe (↑ idp) x
-  => λ i => ↑ primCoe (λ j => A) (~ i) x
+  => λ i => (λ j => A).primCoe x freeze (~ i)
```

Comment:
Is this a bug?

Reply:
![image](https://user-images.githubusercontent.com/16398479/200152861-55ff5620-b314-4401-9bf9-c0a05ed139c6.png)

Reply:
I still think it is abnormal... I will try giving a counterexample without using primitives.

Reply:
:thinking: